binding.cc

```diff
@@ -148,14 +148,16 @@ namespace zmq {
 
       class IncomingMessage;
       static NAN_METHOD(Recv);
+      static NAN_METHOD(Readv);
       class OutgoingMessage;
       static NAN_METHOD(Send);
+      static NAN_METHOD(Sendv);
       void Close();
       static NAN_METHOD(Close);
 
       Nan::Persistent<Object> context_;
       void *socket_;
-      int32_t pending_;
+      bool pending_;
       uint8_t state_;
       int32_t endpoints;
```

Comment:
This is a private member, so it should have a trailing underscore to conform with the rest. I'll just leave this comment here for future reference.